modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/action/ViewEntryMVCRenderCommand.java

```diff
@@ -71,6 +73,13 @@ public String render(
 		try {
 			BlogsEntry entry = ActionUtil.getEntry(renderRequest);
 
+			HttpServletRequest httpServletRequest =
+				_portal.getHttpServletRequest(renderRequest);
+
+			_assetEntryService.incrementViewCounter(
+				BlogsEntryAssetEntryUtil.getAssetEntry(
+					httpServletRequest, entry));
+
```

Comment:
This should only apply when viewed through the regular blogs portlet, not the admin apps.

Reply:
Also, I believe this won't cover the case when an entry is viewed through a mapped field in a display page template. Can you verify this point?

Reply:
> Also, I believe this won't cover the case when an entry is viewed through a mapped field in a display page template. Can you verify this point?

In what concern the display page template, I think it's what this whole [LPP-48215](https://issues.liferay.com/browse/LPP-48215) is all about, I've tested those implementations and they worked as expected, fixing the issue.


> This should only apply when viewed through the regular blogs portlet, not the admin apps.

Could you please elaborate more? I'm little familiarized with the blogs component, therefore I do not know some rules that involves it.

Reply:
@adolfopa

Reply:
> In what concern the display page template, I think it's what this whole LPP-48215 is all about, I've tested those implementations and they worked as expected, fixing the issue. Yet, I'll take another look at it just in case I let something pass.

Hi @adolfopa ! I've tested the view counter increment on a display page template and it worked.

Reply:
About blogs admin vs blogs, this action is invoked when viewing blog entries both from public pages (where we want to increase the view count) and from the administrative apps (where we shouldn't increase the count).

The simplest way to exclude the admin portlet is simply to check the portlet ID. Something like:
```
if (!Objects.equals(portletDisplay, BlogsPortletKeys.BLOGS_ADMIN)) {
    ... increase the view count ...
}
```

Reply:
Hi @adolfopa, thanks for the clarification, I see the problem now, I'll proceed to correct it! 😄